src/components/Player/Header/PlayerStats.jsx

```diff
@@ -46,7 +46,7 @@ export const PlayerStatsCards = ({
         />
         <CardTitle
           className={styles.playerStats}
-          subtitle={`${((wins / (wins + losses)) * 100).toFixed(2)}%`}
+          subtitle={wins + losses ? `${((wins / (wins + losses)) * 100).toFixed(2)}%` : 'N/A'}
```

Comment:
I think there's a N/A string in the strings file to use, otherwise can you put it in en-US.json?